tests/containers/bci_test.pm

```diff
@@ -33,13 +33,15 @@ sub parse_logs {
     my $root = $dom->createElement('testsuites');
     $dom->setDocumentElement($root);
 
-    record_info('Files', script_output('ls -lh'));
+    my $result_files = script_output('ls | grep junit');
```

Comment:
These files will only appear *after* you ran the tests.

Reply:
Yes, and this method is called at the very end after all the tox commands.